docs/tutorials/measurement.md

```diff
@@ -1,105 +1,113 @@
-# 測定
+# Measurement with CARET
 
-サンプルのアプリケーションを例に、ツールの使い方を説明していきます。
+This page explains usage of CARET with a sample application.
+The sample application is located on [CARET_demos](https://github.com/tier4/CARET_demos.git) repository.
 
-サンプルのリポジトリ<https://github.com/tier4/CARET_demos.git>
+## Building application with CARET
 
-## アプリケーションのビルド
+To measure target applications' performance, the target should be built with CARET/rclcpp. CARET/rclcpp is another implementation of [ROS 2-based rclcpp](https://github.com/ros2/rclcpp) which has some additional tracepoints defined by CARET. If you have already built the target without CARET/rclcpp, you have to build the target with CARET/rclcpp.
```

Comment:
How about change the second sentence to the following?
> CARET/rclcpp is fork of [ROS 2-original rclcpp](https://github.com/ros2/rclcpp)

Reply:
fixed in https://github.com/tier4/CARET_doc/pull/35/commits/633a4b3a98e2ed2bb1ed2770ba0745f1c1e79be4